server/src/main/java/org/apache/druid/server/coordinator/helper/DruidCoordinatorSegmentCompactor.java

```diff
@@ -160,7 +160,11 @@ private CoordinatorStats doRun(
             config.getTuningConfig(),
             config.getTaskContext()
         );
-        LOG.info("Submitted a compactTask[%s] for segments[%s]", taskId, segmentsToCompact);
+        LOG.info(
+            "Submitted a compactTask[%s] for segments[%s]",
+            taskId,
+            segmentsToCompact.stream().map(DataSegment::getIdentifier).collect(Collectors.joining(","))
```

Comment:
This joiner thing looks like it would be a good utility function. Including a space after the comma would be a good idea, too, I think, since it visually spaces them out a bit more and would make the log line easier on the eyes.